ARnft/arNFT_intereactive_example.html

```diff
@@ -38,6 +38,9 @@
                     let canvas = document.getElementById('canvas');
                     let fov = 0.8 * 180 / Math.PI;
                     let ratio = window.clientWidth / window.clientHeight;
```

Comment:
This was causing the issue, the result of `window.clientWidth / window.clientHeight` is NaN see the message in the console:
```
ARnft  0.13.1
ARnft.ts:204 ARnft init() start...
arNFT_intereactive_example.html:41 NaN
arNFT_intereactive_example.html:43 1.3333333333333333
SceneRendererTJS.ts:84 ARnftThreejs version:  0.4.5
arNFT_intereactive_example.html:69 
```
and this was the reason of the bad behaviours.

Reply:
fixed with https://github.com/kalwalt/kalwalt-interactivity-AR/pull/46/commits/16d33cedd470efc0ed6905cfe1b8a8f6c3e9e6fb